srm_oparray.c

```diff
@@ -252,6 +268,11 @@ static const op_usage opcodes[] = {
 	/*  193 */	{ "GET_TYPE", ALL_USED },
 	/*  194 */	{ "FUNC_NUM_ARGS", ALL_USED },
 	/*  195 */	{ "FUNC_GET_ARGS", ALL_USED },
+	/*  196 */	{ "UNSET_CV", ALL_USED },
+	/*  197 */	{ "ISSET_ISEMPTY_CV", ALL_USED },
+#endif
+#if PHP_VERSION_ID >= 70300
+	/*  198 */  { "FETCH_LIST_W", ALL_USED },
```

Comment:
Would you mind using the same white space as in the 197 line please?

Reply:
I'm sorry about it, fixed in my next commit